conf/messages

```diff
@@ -173,5 +173,7 @@ one_login_complaint.radio.letter-message.label=Letter
 one_login_complaint.complaint.label=Complaint (optional)
 one_login_complaint.complaint.hint=What problems are you experiencing and how can we help you. Please provide all the relevant information in the box below.
 one_login_complaint.complaint.error=Complaint cannot be longer than 1000 characters
-one_login_complaint.confirmation.title=One Login for Government complaint – GOV.UK
+one_login_complaint.confirmation.title=We have received your One Login for Government complaint
```

Comment:
does this need the ` – GOV.UK` maintained, or is it added somewhere else and it being in here before was duplication?

Reply:
@oscarduignan @PatrykRudnicki I think it needs  `– GOV.UK` maintained

Reply:
it doesn't say definitively I guess, just says "can have" https://design.tax.service.gov.uk/hmrc-design-patterns/page-title/ but I'd include it I think on all pages

Reply:
it's something that I know that dias flag up in audits

Reply:
Added to confirmation screen title